core50.md

```diff
@@ -0,0 +1,237 @@
+# Apache MyFaces Core 5.0
+
+Implementation of the Jakarta Faces 5.0 specification.  This is still in development and not yet released. 
+
+## Requirements
+* Java 21
+* Servlet 6.1+
+* EL 6.0
+* CDI 4.1
+* JSTL 3.0 (optional)
+* BV 3.1 (optional)
```

Comment:
```suggestion
* Java  11 (EE Platform: 21) 
* Servlet 6.0 (EE Platform: 6.1)
* EL 5.0 (EE Platform: 6.0)
* CDI 4.0 (EE Platform: 4.1)
* Tags (EE Platform: 3.0) (optional)
* BeanValidation 3.0 (EE Platform: 3.1) (optional)
```

Reply:
@tandraschko  Something like this?  Perhaps we should use Java 17, but I can start a mailing list discussion about this.

Reply:
@tandraschko Can you comment? See the emails here: https://www.eclipse.org/lists/faces-dev/msg00354.html

There's a push to move to 17 or 21.

Reply:
+0 tbh
As long we dont need to up, we can simply stay on java11